### PR TITLE
alphabetizing the language files

### PR DESCRIPTION
**Pull request by trigal2012**
## Description

the language file keys were getting out of sync with one another and not all were alphabetized. This PR entailed only re-ordering the keys across all files getting them in sync again.

## PR Checklist

- [x] Assign @FJKhan **and** @Alfredo-Moreira as reviewers.
- [x] If your PR is not a hotfix, is it targeted for `dev`? If it is a hotfix, is it targeted for `main`?
- [x] Unit and functional test coverage was added where applicable.
- [x] CI/CD passes for your PR.
- [x] Complex code is well documented with comments.
- [x] Does the original ticket have test instructions? If not add them below
- [x] Pass QA Gate(manual testing)

### CODE REVIEW

**Comment by trigal2012** [comment]
@Alfredo-Moreira @FJKhan can one of you take a look at this PR? super simple, just alphabetizing the language files. trying to keep them organized as I add new keys to it.

File: src/lang/en-CA.json [hunk 1]
```diff
@@ -1,356 +1,356 @@
...
-	"account.no-account": "Don't have an account?",
-	"account.request-password-reset": "Enter your email address and we will send you an email to reset your password.",
-	"account.sign-out": "Sign Out",
-	"account.sign-up": "Sign Up",
-	"account.sign-in": "Sign In",
 	"account.already-have-account": "Already have an account?",
+	"account.already-organisation-affiliate": "You are already affiliated with [name]",
+	"account.finish-registration": "Finish Registration",
+	"account.forgot-password": "Forgot Password?",
 	"account.join-organisation": "Join Your Organization",
```
Comment (Alfredo-Moreira) [inline src/lang/en-CA.json]: why are all these organization references spelled organisation?

**Comment by Alfredo-Moreira** [comment]
Besides the organization's spelling everything looks good.

**Review by Alfredo-Moreira — CHANGES_REQUESTED** [review]

File: src/lang/en-CA.json (hunk 1 above)
Comment (trigal2012) [reply src/lang/en-CA.json]: I noticed that there are several places in the language files where the key has organization spelled as 'organisation' such as in the example you mentioned. Being that it is the key and not the value of what gets displayed in the UI, I did not update it to 'organization'. 

I can change all of the keys for consistency. BTW, I'll be making more changes in a future PR.

**Comment by trigal2012** [comment]
I'm closing this PR and will roll the changes here into the sign up flow change PR.

**Pull request closed without merging**